script/events/encounters.js

```diff
@@ -396,5 +396,41 @@ Events.Encounters = [
 				notification: _('a shot rings out, from somewhere in the long grass')
 			}
 		}
+	},
+	{
+		title: _("A madman"),
```

Comment:
Reasoning to add a new encounter?

maybe turn this to an earlier encounter that's easier but provides a `i knife`